examples/wallet/package.json

```diff
@@ -17,6 +18,7 @@
         "@vitejs/plugin-react-swc": "^3.4.0",
         "eslint": "^8.50.0",
         "typescript": "^5.2.2",
+        "ts-node": "10.9",
```

Comment:
Why is this needed? For vite to understand config files in typescript or?

Reply:
Exactly, didn't want to start a fight with the entire configuration, so I just added it here